### DISCUSSION TITLE
A Payment method is required to pay failed payment error

### DISCUSSION THREAD

**Discussion by myperfectfit-dev** [vercel/community#4514] [Account Issue]
### Summary

Whats happening now i cant pay as system is not all seems to check the card.. the same card is used to pay using the other account, Kindly generate new link for the payment and send us. This is a small issue from your end. Please address fast as its been 2 days we are still waiting

![image](https://github.com/vercel/community/assets/75773373/96b69c17-6cf3-4994-aa65-566bbdcf87f2)

### Steps to Reproduce

Click on the payment method it saying payment method required to do failed payment. System error

**Comment by amyegan** [comment]
I understand you are eager for a solution to your problem, however we cannot safely discuss payment information here in public. More information and instructions were provided in your support ticket. Please continue to follow up with that ticket. 

Closing this discussion as a duplicate of https://github.com/orgs/vercel/discussions/4507
